spring-integration-jms/src/main/java/org/springframework/integration/jms/JmsHeaders.java

```diff
@@ -19,9 +19,13 @@
 /**
  * Pre-defined names and prefixes to be used for setting and/or retrieving JMS
  * attributes from/to integration Message Headers.
- * 
+ *
+ * @deprecated - use {@link org.springframework.jms.support.JmsHeaders.JmsHeaders}.
```

Comment:
Double `JmsHeaders` causes the `api` task failure.
Also it shows how are we wrong with the `public DefaultScriptExecutor` JavaDocs. :smile: 
Will fix on merge.